person.rb

```diff
@@ -0,0 +1,28 @@
+require_relative 'nameable'
+
+class Person < Nameable
+  attr_reader :id, :rentals
```

Comment:
- [ ] Likewise as mentioned in a comment above, kindly set both the getter and setter for the :rentals to implement the "A person has many rentals" functionality correctly. This should be like this: ```attr_accessor :name, :age, :rentals```

Reply:
Fixed.